src/client/eppo-client.ts

```diff
@@ -209,12 +212,15 @@ export default class EppoClient {
    * @param defaultValue default value to return if the subject is not part of the experiment sample
    * @returns a boolean variation value if the subject is part of the experiment sample, otherwise the default value
    */
-  public getBooleanAssignment = (
+  public getBooleanAssignment(
```

Comment:
arrow functions caused a few TypeScript errors in `EppoJSClient`, so I reverted back to class methods